src/nile/utils/__init__.py

```diff
@@ -127,12 +130,21 @@ def normalize_number(number):
 def hex_address(number):
     """Return the 64 hexadecimal characters length address."""
     if type(number) == str and number.startswith("0x"):
-        return _pad_hex_to_64(number)
+        return _pad_hex_to(number, ADDRESS_LEN)
+    else:
+        return _pad_hex_to(hex(int(number)), ADDRESS_LEN)
+
+
+def hex_class_hash(number):
+    """Return the 65 hexadecimal characters length class hash."""
```

Comment:
Either change the number here to 63 or change the comment in hex_address's number to 66 to keep consistency (I didn't consider the "0x" prefix as part of the length of the hexadecimal characters when I added the comment).

Reply:
+1 to the former

Reply:
the current implementation returns a 67 char len string, 0x + 65 chars. The constants at the beginning of the file refer to `n` in the the `0x + n` count. I think the comment is misleading, it should say the full length, including the 0x.

therefore this comment should say 67, and `hex_address`'s 66